python-sdk/tests_integration/databases/test_mysql.py

```diff
@@ -0,0 +1,132 @@
+"""Tests specific to the MySql Database implementation."""
+import pathlib
+import re
+from urllib.parse import urlparse
+
+import pandas as pd
+import pytest
+import sqlalchemy
+
+from astro.constants import Database
+from astro.databases import create_database
+from astro.databases.mysql import MysqlDatabase
+from astro.settings import SCHEMA
+from astro.table import Metadata, Table
+
+DEFAULT_CONN_ID = "mysql_default"
+CUSTOM_CONN_ID = "mysql_conn"
+SUPPORTED_CONN_IDS = [DEFAULT_CONN_ID, CUSTOM_CONN_ID]
+CWD = pathlib.Path(__file__).parent
+
+TEST_TABLE = Table()
+
+
+@pytest.mark.parametrize("conn_id", SUPPORTED_CONN_IDS)
+def test_create_database(conn_id):
```

Comment:
@phanikumv Can we move this test to Unit test section?

Reply:
done